clients/drcachesim/tools/schedule_stats.cpp

```diff
@@ -477,6 +492,14 @@ schedule_stats_t::aggregate_results(counters_t &total)
         assert(shard.second->counters.direct_switches ==
                shard.second->stream->get_schedule_statistic(
                    memtrace_stream_t::SCHED_STAT_DIRECT_SWITCH_SUCCESSES));
+        // If the scheduler is counting migrations, it may see more due to inputs
+        // not yet executed moving among runqueues.
+#ifndef NDEBUG
+        double sched_migrations = shard.second->stream->get_schedule_statistic(
+            memtrace_stream_t::SCHED_STAT_MIGRATIONS);
+#endif
+        assert(sched_migrations == 0. ||
+               sched_migrations > shard.second->counters.observed_migrations);
```

Comment:
The comment says it "may" see more, but the assertion seems to require it to see more.  Should that be >= instead of > in the assert()?